h/static/scripts/vendor/jquery.scrollintoview.js

```diff
@@ -109,29 +109,37 @@
 
 				var animOptions = {};
 
+				var yDir = "none";
```

Comment:
> > However, it's not only the "scrolling" which is problematic in those situations; the whole thing needs a full rethinking.
> 
> Exactly. I want to remove it all and find something better. I don't want something 80% in the meantime.
> I am advocating for the complete removal of the bucket bar and have for a long, long time.

I have no objection against (trying to) replace this (or any other) feature with something better, but how about doing it in the opposite order?
 1. Let's start to run experiments and try to come up with something better.
 2. When we have something consistently better, replace our current solution with that.

Just ripping out our current solution (which, if you ask me, is closer to 99.9% than to 80%), without providing a replacement, feels like kicking our users *and ourselves* in the groin.